package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@oat-sa/tao-core-ui",
-    "version": "1.22.9",
+    "version": "1.22.10",
```

Comment:
This is no longer needed, correct?

Reply:
The bump version is required since it's `npm` package.